react-validated-input-component/src/validated-input.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+class ValidatedInput extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { password: '', errorMsg: '' };
```

Comment:
The `errorMsg` state doesn't look like it's being used, so I'd suggest getting rid of it.

Reply:
Noted and corrected